profileAbout.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="prostyle.css">
+    <title>About</title>
+  </head>
+
+  <main class="backDrop">
+
+  <body>
```

Comment:
The body tag should not be used more than once and shouldn't be nested inside any tag other than html. All the content on a page should be inside the body. For example:

<html>
    <body>
        All the page elements and content goes in here.
    </body> 
</html>